tools/generator/Java.Interop.Tools.Generator.CodeGeneration/CodeGenerator.cs

```diff
@@ -914,8 +922,11 @@ public void WriteInterfaceListenerEventOrProperty (InterfaceGen @interface, Meth
 				if (opt.GetSafeIdentifier (name) != name) {
 					Report.Warning (0, Report.WarningInterfaceGen + 5, "event name for {0}.{1} is invalid. `eventName' or `argsType` can be used to assign a valid member name.", @interface.FullName, name);
 					return;
-				} else
-					WriteInterfaceListenerEvent (@interface, indent, name, nameSpec, m.AdjustedName, full_delegate_name, !m.Parameters.HasSender, connector_fmt, add, remove);
+				} else {
+					var mt = target.Methods.Where(method => string.Compare(method.Name, connector_fmt, StringComparison.OrdinalIgnoreCase) == 0 && method.IsListenerConnector).FirstOrDefault();
```

Comment:
Thanks for pointing it out, all style fixed.
I am still getting used to this new coding style. All other teams I worked at Microsoft used the StyleCop coding style, would be nice if we catch all those style errors during build time.

Reply:
We have an `.editorconfig`, so that should kick in if you're using something like VS: https://github.com/xamarin/java.interop/blob/master/.editorconfig.  (But yeah, Mono has always had a very "unique" coding style.)  :)

Reply:
I am using vscode, does it work with vscode?

Reply:
just found this

https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

hopefully it will help me :)